apiserver/restrict_newer_client.go

```diff
@@ -11,32 +11,45 @@ import (
 	jujuversion "github.com/juju/juju/version"
 )
 
-// minAgentMinorVersions defines the minimum minor version
-// for a major agent version making api calls to a controller
-// with a newer major version.
-var minAgentMinorVersions = map[int]int{
-	2: 9,
+// minAgentVersions defines the minimum agent version
+// allowed to make a call to a controller with the major version.
+var minAgentVersions = map[int]version.Number{
+	3: version.MustParse("2.9.32"),
```

Comment:
Why .32 here?

Reply:
cause 2.9.32 *agents* are ok. the recent work in facade stuff was for client facades